build_scripts/rpm/mssql-cli.spec

```diff
@@ -36,8 +36,9 @@ BuildArch:      x86_64
 BuildRequires:  gcc
 BuildRequires:  libffi-devel
 BuildRequires:  openssl-devel
+BuildRequires:  /usr/bin/pathfix.py
 
-Requires:       libunwind, libicu
```

Comment:
Are you sure we don't need this?

Reply:
In my testing, mssql-cli worked just fine without libunwind.

Reply:
Perhaps because it's already installed? If we have no specific reason to remove this, then let's leave it. I think this was needed by the sqltoolsservice as a runtime dependency.